src/util/expr.h

```diff
@@ -37,8 +37,17 @@ class depth_iteratort;
 class const_depth_iteratort;
 class const_unique_depth_iteratort;
 
-/*! \brief Base class for all expressions
-*/
+/// Base class for all expressions.
+/// Inherits from \ref irept and has operands (stored as unnamed children of
```

Comment:
:bulb: Could be a bit newbie-friendlier, e.g. "Inherits from the tree structure irept" or something similar. We don't want to re-explain irept every time it's mentioned, but in it's direct subclasses I think a little bit of duplication won't do any harm.